prowler/providers/aws/services/dms/dms_endpoint_ssl_enabled/dms_endpoint_ssl_enabled.metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Provider": "aws",
+  "CheckID": "dms_endpoint_ssl_enabled",
+  "CheckTitle": "Ensure SSL mode is enabled when creating a DMS endpoint",
+  "CheckType": [
+      "Data Protection",
+      "Encryption in Transit"
```

Comment:
This type should be extracted from: https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#Types